### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: Store.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  delete() {
+    return this._engine.delete();
+  }
+
+  get() {
+    return this._engine.get();
+  }
+
+  id() {
+    return this.engine.id();
```
Comment (dlongley) [inline Store.js]: Missing underscore.

File: Store.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import MemoryEngine from './engines/MemoryEngine';
```
Comment (dlongley) [inline Store.js]: `./engines/MemoryEngine.js`

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+Bedrock Non-Commercial License v1.0
+===================================
+
+Copyright (c) 2011-2016 Digital Bazaar, Inc.
```
Comment (mattcollier) [inline LICENSE.md]: bump date

**Review by dlongley — CHANGES_REQUESTED** [review]
I'm going to push a commit up with some changes to the design here.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "bedrock-web-store",
+  "version": "0.1.0",
+  "description": "Shared storage for Bedrock Web Apps",
+  "main": "index.js",
+  "dependencies": {
+    "pure-uuid": "^1.5.2"
```
Comment (mattcollier) [inline package.json]: Is there something wrong with the uuid dep we've been using?  Here's a usage example:  

https://github.com/digitalbazaar/veres-delta/blob/cm4/package.json#L101
https://github.com/digitalbazaar/veres-delta/blob/cm4/components/ledger-express-config/ledger-creator-component.js#L5

Aside from consistency, the API for this new lib is nasty.

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (dlongley) [reply package.json]: We actually don't need any UUID dependency here I don't think. But, yes, there is a problem with the UUID dep we've been using. It only works on node -- or its "browser" version is broken/difficult to use in some fashion, likely in its build mechanism.

**Review by dlongley — APPROVED** [review]

**Pull request merged**